src/shell.ts

```diff
@@ -206,6 +218,26 @@ export class Shell {
             e.classList.add("hidden")
     }
 
+    startWatchdog() {
```

Comment:
Add a function comment as it's a weird function. It never resolves and only occasionally rejects.